### DISCUSSION TITLE
How to add a splash screen

### DISCUSSION THREAD

**Discussion by jlopez994** [cawa-93/vite-electron-builder#963] [Q&A]
Hi,

I'm trying to add a splash screen but I'm not sure how to start with this structure.

Any suggestion?

Thanks!

**Comment by cawa-93** [comment]
I haven't checked, but theoretically, you can create a new package like a renderer with the same configuration. This will allow you to create any visual, just like you do in the renderer. 
Then, in the main, create a new window in the same way as for the renderer, but with different parameters and URIs.
